### PR TITLE
Add cluster_operator e2e test

### PR DESCRIPTION
**Pull request by csrwng**
Adds e2e tests for Cluster Operator

### CODE REVIEW

**Comment by csrwng** [comment]
@dgoodwin fyi
The test is working but failing for right now because of the cert mismatch on the external ELB
It's also pending proper credentials from @dobbymoodge 
Currently it simply uses the ones from jenkins

File: sjb/config/test_cases/test_branch_cluster_operator_e2e.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        sudo yum -y install epel-release-latest-7.noarch.rpm
+        sudo yum --disablerepo="*" --enablerepo=epel -y update ansible
+        sudo env GOPATH=/data go get -u github.com/cloudflare/cfssl/cmd/...
+        mkdir -p /tmp/.aws
+        mkdir -p /tmp/.ssh
+    - type: "host_script"
+      title: "transfer aws credentials to remote host"
+      script: |-
+        scp -F ./.config/origin-ci-tool/inventory/.ssh_config ~/.aws/clusteroperator-credentials openshiftdevel:/tmp/.aws/credentials
+        scp -F ./.config/origin-ci-tool/inventory/.ssh_config ~/.ssh/devenv.pem openshiftdevel:/tmp/.ssh/libra.pem
```
Comment (dobbymoodge) [inline sjb/config/test_cases/test_branch_cluster_operator_e2e.yml]: Isn't the SSH pubkey added to the instance by the cloud provider?

Comment (dobbymoodge) [reply sjb/config/test_cases/test_branch_cluster_operator_e2e.yml]: Also, maybe for these instances we should use an ad-hoc or less widely shared keypair. Thoughts?

Comment (csrwng) [reply sjb/config/test_cases/test_branch_cluster_operator_e2e.yml]: @dobbymoodge the public key is added to the instance, but not the private one.
We need the private key in the test vm so *it* can ssh into the target cluster machines.

If you want to create a new key just for this, that's great. I'd only ask that you let me know the name and to place the private key somewhere on the jenkins filesystem so the job can get to it.

**Comment by csrwng** [comment]
@dobbymoodge I've now updated the job to create a keypair and use that for the target cluster's machines.

**Comment by csrwng** [comment]
@dobbymoodge updated to remove bash calls with sudo, got one passing test. It's ready to ship! :)

**Pull request merged**